### ISSUE TITLE
Improve API documentation with information about limit and offset

### ISSUE THREAD

**Issue by fernandobarbalho** [okfn-brasil/serenata-de-amor#494]
The API documentation indicates that to list all embursments you have to execute the following get command

Listing reimbursements
GET /api/chamber_of_deputies/reimbursement/
Lists all reimbursements.

althoug, when we execute this command line the API retrives only the seven first rows. It is necessary to also indicate parameters limit and offset in order to fetch through the database, as I did in my code, as indicated bellow (R code).

```
library(jsonlite)

reembolso<- fromJSON("https://jarbas.serenata.ai/api/chamber_of_deputies/reimbursement/?limit=20&offset=20")

#LIsts 20 congressperson names from the 21st position 
reembolso$results$congressperson_name

```

**Comment by cuducos** [comment]
> It is necessary to also indicate parameters limit and offset in order to fetch through the database

TLDR My take on that is that: no, it's not necessary.

The _next page_ URL (including `limit` and `offset`)  is given in the `next` (*or* `previous`, when it's available) fields of every response. So users doesn't have to do the math, just use the URL given by the API itself. 

<img width="942" alt="Pasted_Image_15_09_19_10_50" src="https://user-images.githubusercontent.com/4732915/64923321-d46d8100-d7a6-11e9-86fe-d270e720452a.png">

This adopted largely in REST APIs and is important: it allows maintainers of the project to (try to) keep control of the `offset` and `limit` thinking about PostgreSQL performance and backend limitations (e.g. maybe override user's `limit` to avoid a query too big that might disrupt the service). So, as in the REST principle, following the API response to go to the next page: it's good to the users (the link is already there, as precise as it needs to be), it's good to the maintainers (this helps us keeping Jarbas alive).

**Comment by fernandobarbalho** [comment]
Great. Next and previous really help a lot. 

Maybe this explanation you've written could be added to the documenation.
